sailthru/sailthru_import/test_sailthru_import_verify.py

```diff
@@ -1,51 +0,0 @@
-import os
```

Comment:
This is a poor attempt at testing this import script. The whole file is full of bugs. It's not useful so we remove it. It we need this later, we can code a better one